### ISSUE TITLE
Potential impacts of losing .io domains

### ISSUE THREAD

**Issue by BenTheElder** [etcd-io/etcd#18744]
### What would you like to be added?

See previous discussion about the possible problem in https://github.com/kubernetes/kubernetes/issues/127966

We may want to consider at least reserving an alternate domain. 

I've filed CNCFSD-2503 to ask the CNCF about reserving another TLD (just to reserve, with my SIG K8s Infra hat on, the project is free to use it however if we acquire one, but I'd recommend setting it up as a 1:1 backup, SIG K8s Infra can help with that). etcd.dev seems to be taken already, but maybe we can acquire something else.

### Why is this needed?

Because IO will likely stop being a country code soon, and without an exception would start a process to wind down the ccTLD. Even if that doesn't happen, some concerns have been expressed about .io (see linked threads).

IMHO: Do not panic, nothing is happening yet and even if something does happen there will be time to respond. However, it is pretty easy for SIG K8s Infra to manage domains, and it seems wise to go ahead and park an alternate.

**Comment by BenTheElder** [comment]
Got a response on the ticket already, TLDR: CNCF are aware and putting together a plan before registering any more domains.

**Comment by ahrtr** [comment]
Thanks @BenTheElder .

Unfortunately, all relevant & good alternatives, which I can think of, are already taken,
- etcd.dev
- etcd.tech
- etcd.cloud

Not sure whether it's possible to request CNCF try to spend some money to acquire one of them.

**Comment by BenTheElder** [comment]
> Not sure whether it's possible to request CNCF try to spend some money to acquire one of them.

I have an open CNCF service desk ticket (CNCFSD-2503 from above) tracking this topic.

They're making a general all-projects plan for this problem space and we're waiting on that currently before making any more requests.

**Comment by jberkus** [comment]
LOL.  Someone has put up etcd.org for sale for $6500.

**Comment by ahrtr** [comment]
> LOL. Someone has put up etcd.org for sale for $6500.

Ah, yes. Could we request CNCF to acquire the domain?

**Comment by BenTheElder** [comment]
> Ah, yes. Could we request CNCF to acquire the domain?

We can, just recently got follow-up about this. If we have specific domain requests we can send them through the service desk (they followed up on the previous issue related to this).

**Comment by BenTheElder** [comment]
See also https://github.com/kubernetes/kubernetes/issues/127966#issuecomment-2884716831

**Comment by williamdes** [comment]
Keep it

**Comment by siyuanfoundation** [comment]
no news about this yet. 
Keep watching.

**Comment by jberkus** [comment]
In the last 15 months, IANA and Mauritus have not announced any disposition of the IO domain, so this is on hold.  In the meantime, several Etcd maintainers have acquired alternate domains we can use if required.

**Comment by ivanvc** [comment]
@jberkus, is this list of alternate domains documented somewhere?
